### DISCUSSION TITLE
Where  1=0

### DISCUSSION THREAD

**Discussion by christopher-kleine** [aarondl/sqlboiler#1297] [Q&A]
Hi!

I couldn't find a solution to this, so if this is a duplicate, let me know!

I'm using I have two tables:
```sql
CREATE TABLE users (
	id   VARCHAR NOT NULL,
	name VARCHAR NOT NULL,

	PRIMARY KEY (id)
);

CREATE TABLE items (
	id      VARCHAR NOT NULL,
	name    VARCHAR NOT NULL,
	user_id VARCHAR NOT NULL,

	PRIMARY KEY (id),
	FOREIGN KEY (user_id) REFERENCES users
);
```

If I want to load all items into a user, I can run:
``` go
users, err := models.Users(
	models.UsersWhere.ID.EQ(id),
	qm.Load(models.UserRels.Items),
).All(ctx, dbconn)
```

This works fine.
But if a user has no items, SQLBoiler creates this query for the items:
```sql
SELECT "items".* FROM "items" WHERE (1=0);
``` 

Wouldn't it be more efficient to simply skip it?
If this is decided by the configuration, I'd like to know. Since - as I mentioned - I couldn't find anything about it.

Have a nice day!

**Comment by stephenafamo** [comment]
At first thought, this looks like a bug. Ideally, it should be skipped.  

Can you open an issue with all the required details in the issue template 🙏🏾
